.grabl/automation.yml

```diff
@@ -135,6 +135,9 @@ build:
         sudo systemctl daemon-reload
         sudo systemctl start typedb
 
+        pip3 install -U pip
+        pip install typedb-client==2.2.0
```

Comment:
Previously we loaded in client-python via Bazel, which is a remnant from client-python 1.8. We don't do that anymore.

It was working fine for a while, and then unexpectedly broke causing "metadata leak" errors in CI. So now we're using the recommended installation method via `pip`.